x-pack/plugins/reporting/export_types/printable_pdf/server/lib/layouts/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export { getlayout } from './layout_factory';
+export { Optimizedlayout } from './optimized_layout';
```

Comment:
I'd keep this named `PrintLayout`.  While both are pretty confusing, I think `print` is less ambiguous than `optimize` (which makes me think, it's doing something more performant).  And it fits with the current code more.